### PR TITLE
feat(locksmith) - make lock name required to included it in email

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
This PR makes `lockName` required to always include it into the email.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/controllers/v2/ticketsController.ts [hunk 1]
```diff
@@ -108,11 +109,15 @@ export class TicketsController {
         keyId,
         network
       )
+
+      const lock: Lock = await this.web3Service.getLock(lockAddress, network)
```
Comment (kalidiagne) [inline locksmith/src/controllers/v2/ticketsController.ts:113]: @searchableguy we have a more efficient way to get it? just need to get the lockName without passing it as a parameter

**Review by searchableguy — APPROVED** [review]
LGTM

**Review by julien51 — APPROVED** [review]
Lgtm

**Comment by julien51** [comment]
Please check why the test fails?

**Pull request merged**